patches/zotero2readwise/readwise.py

```
@@ -0,0 +1,464 @@
+
+        last_result = None
+        for batch_num, i in enumerate(range(0, total, batch_size), start=1):
+            batch = highlights[i : i + batch_size]
+            print(
+                f"  Uploading batch {batch_num}/{num_batches} "
+                f"({i + 1}–{min(i + batch_size, total)} of {total})...",
+                flush=True,
+            )
+
+            for attempt in range(1, max_retries + 1):
+                resp = requests.post(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)